doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,178 @@
+Zachary Pincus. The SC membership is revisited every January. SC members who do
+not actively engage with the SC duties are expected to resign. New members are
+added by nomination by a core developer. Nominees should have demonstrated
+long-term, continued commitment to the project and its :ref:`values`. A
+nomination will result in discussion that cannot take more than a month and
+then admission to the SC by consensus.
+
+
+Decision Making Process
+=======================
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project `mailing list <mailto:scikit-image@python.org>`_
+and the `issue tracker <https://github.com/scikit-image/scikit-image/issues>`_.
+Occasionally, sensitive discussion may occur on a private list.
+
+Decisions should be made in accordance with the :ref:`mission, vision and
+values <values>` of the scikit-image project.
+
+Scikit-image uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+Core developers are expected to distinguish between fundamental objections to a
+proposal and minor perceived flaws that they can live with, and not hold up the
+decision-making process for the latter.  If no option can be found without
+objections, the decision is escalated to the SC, which will itself use
+consensus seeking to come to a resolution. In the unlikely event that there is
+still a deadlock, the proposal will move forward if it has the support of a
+simple majority of the SC. Any vote must be backed by a :ref:`scikit-image
+proposal (SKIP) <skip>`.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence (but no change of the scikit-image.org landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus).
+
+- **Changes to the API principles** require a :ref:`skip` and follow the
+  decision-making process outlined above.
+
+- **Changes to this governance model or our mission, vision, and values**
+  require a :ref:`skip` and follow the decision-making process outlined above,
+  *unless* there is unanimous agreement from core developers on the change.
+
+If a veto is cast on a lazy consensus, the proposer can appeal to the
```

Comment:
This is the only mention of the word "veto" in this PR. It's not clear what the definition is. Could be interpreted as anything that's not "lazy consensus" (which is defined), but that doesn't really make sense (one person saying "I'm not 100% sure this is the best idea" `!=` "veto"). I suggest to reword or explain.

The rest of this PR looks really good!